### PR TITLE
Build

### PR DESCRIPTION
**Pull request by Mav7**
@jazzonmymind @zzak Sorry it's been awhile I've been busy with my other project and trying to find a job. I wrote new docs for mrb_p(), mrb_obj_id(), mrb_obj_to_sym(), mrb_obj_eq(), mrb_obj_equal(), mrb_equal(), mrb_convert_to_integer(), mrb_Integer(), mrb_Float(), mrb_inspect(), mrb_eql(). Please review! :)

### CODE REVIEW

**Comment by matz** [comment]
Why did you know close the issue?

**Comment by Mav7** [comment]
@matz Sorry I meant to PR this on my fork first to it could review there before bringing to the main project here.

**Comment by matz** [comment]
Never mind. I will review this here anyway.

**Comment by Mav7** [comment]
@matz Oh ok great! :)

File: build_config.rb [hunk 1]
```diff
@@ -21,6 +21,8 @@
 
   # include the default GEMs
   conf.gembox 'default'
+  
+  conf.gem '../mruby_example' # git clone https://github.com/Mav7/mruby_example.git
```
Comment (syohex) [inline build_config.rb]: This change is not necessary. Please remove this change.

File: build_config.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@
 
   # include the default GEMs
   conf.gembox 'default'
+
```
Comment (syohex) [inline build_config.rb]: A new line remains.

**Comment by syohex** [comment]
@Mav7  I suppose you should squash commits before merging. There are some unnecessary commits.

**Comment by Mav7** [comment]
@syohex Ok but I never done this before how would I do this?

**Comment by matz** [comment]
Google git rebase.

**Comment by syohex** [comment]
- https://help.github.com/articles/about-git-rebase/
- https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history
- http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit

**Comment by Mav7** [comment]
@syohex I squashed my commits into one. I think I did it correctly.

**Comment by syohex** [comment]
> Is that correct?

No. Your commits increased before you rebased.

I suppose you should recreate PR.

**Comment by Mav7** [comment]
@syohex Ok so should I close this one and then PR again?

**Comment by syohex** [comment]
Yes, however I suppose your branch is broken by wrong `git rebase` operations. If so, you re-create branch and apply your changes to new branch and pull request again.

**Comment by Mav7** [comment]
@syohex Ok thank you so much!

**Pull request closed without merging**